src/main/java/com/github/onsdigital/babbage/search/external/requests/search/requests/ContentQuery.java

```diff
@@ -96,4 +101,43 @@ protected ContentResponse getContentResponse() throws Exception {
                 .content(new StringContentProvider(MAPPER.writeValueAsString(content)));
         return request.send();
     }
+
+    @Override
+    public SearchResult call() throws Exception {
+        SearchResult result = super.call();
+
+        if (Configuration.SEARCH_SERVICE.EXTERNAL_SPELLCHECK_ENABLED) {
+            // Execute a spell checker request and add to response
+
+            SpellCheckRequest spellCheckRequest = new SpellCheckRequest(super.searchTerm);
+            SpellCheckResult spellCheckResult = spellCheckRequest.call();
+
+            StringBuilder spellingStringBuilder = new StringBuilder();
```

Comment:
Consider pulling the parsing code below out into its own method - personally I think code is easier to read and test when each method does 1 thing. Speaking of tests...